### PR TITLE
US 7 Play-Card

### PR DESCRIPTION
**Pull request by milodevhh**
-  add a function that track the selected cards
-  use Math.floor to check vertical or horizontal
-  add a text and an SVG file to display the game success
-  add a function to restart the game grid
-  add a restart button
-  add styling

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,25 +1,64 @@
...
+    shuffledCards[2]?.isActive &&
+    shuffledCards[5]?.isActive &&
+    shuffledCards[8]?.isActive;
+  const bingo =
+    firstHorizontalRowWin ||
+    secondHorizontalRowWin ||
+    thirdHorizontalRowWin ||
+    firstHorizontalColumWin ||
+    secondHorizontalColumWin ||
+    thirdHorizontalColumWin;
```
Comment (F-Kirchhoff) [inline pages/index.js:42]: okay thats one way :D maybe in the future you find a shorter solution. For now its fine 👍

File: pages/index.js [hunk 2]
```diff
@@ -1,25 +1,64 @@
...
         cards={shuffledCards}
         handleActiveCards={handleActiveCards}
       />
-      <StyledButton onClick={shuffle}>Create new Bingo</StyledButton>
+      <StyledButton onClick={shuffle}>
+        {bingo ? "Restart the game" : "Create new Bingo"}
+      </StyledButton>
+
+      {bingo ? <WinIcon color="" width="3rem" height="3rem" /> : null}
+      {bingo ? <StyledWinText>Congratulations you won!</StyledWinText> : null}
```
Comment (F-Kirchhoff) [inline pages/index.js]: you can combine both elements into one conditional:
```js
      {bingo && <>
         <WinIcon color="" width="3rem" height="3rem" />
         <StyledWinText>Congratulations you won!</StyledWinText>
       </>
}
```

**Review by F-Kirchhoff — APPROVED** [review]
Hey Milo,
you did it :D you created a winning algorithm. I added one little suggestion, not necessary but maybe you want to implment it.

File: pages/index.js (hunk 1 above)
Comment (F-Kirchhoff) [reply pages/index.js:42]: what about the diagonals?

File: pages/index.js (hunk 2 above)
Comment (milodevhh) [reply pages/index.js]: Thanks

File: pages/index.js (hunk 1 above)
Comment (milodevhh) [reply pages/index.js:42]: Yes, for the time being only vertical or horizontal, I had changed that in the US also so then in the apron. The example still shows diagonal, that is correct. But with 9 cards (3X3 Grind) I find only the two winning possibilities better.

**Pull request merged**